frontend/src/pages/PipelineDetails.tsx

```diff
@@ -434,38 +526,46 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
 
     const [graph, reducedGraph, graphV2] = await this._createGraph(templateString);
 
+    // TBD(jlyaoyuli): If we allow upload v1 version to v2 pipeline or vice versa,
```

Comment:
I think with v1beta1 API, we have been allowing uploading both Argo and IR pipelines, so we can't change the behavior. However, in v2beta1 API, we don't need to allow uploading Argo pipelines, we can only accept IR pipelines. 
Not sure if this is the current implementation thought. @gkcalat to confirm.

Reply:
The current implementation is making v1 and v2 exclusive to avoid mix of Argo and IR under single pipeline (folder).